selftests/functional/test_sysinfo.py

```diff
@@ -70,6 +70,25 @@ def test_sysinfo_disabled(self):
         msg = 'Avocado created sysinfo directory %s:\n%s' % (sysinfo_dir, result)
         self.assertFalse(os.path.isdir(sysinfo_dir), msg)
 
+    def test_sysinfo_html_output(self):
+        os.chdir(BASEDIR)
+        html_output = "{}/output.html".format(self.tmpdir.name)
+        cmd_line = ('{} run --html {} --job-results-dir {} --sysinfo=on '
+                    'passtest.py'.format(AVOCADO, html_output,
+                                         self.tmpdir.name))
+        result = process.run(cmd_line)
+        expected_rc = exit_codes.AVOCADO_ALL_OK
+        self.assertEqual(result.exit_status, expected_rc,
+                         'Avocado did not return rc %d:\n%s' % (expected_rc,
+                                                                result))
+        with open(html_output, 'rt') as fp:
+            output = fp.read()
+
+        # Try to find some strings on HTML
+        self.assertNotEqual(output.find('Filesystem'), -1)
+        self.assertNotEqual(output.find('root='), -1)
```

Comment:
This would probably break when run on non-Linux systems... but, since we've not declared support for non-Linux systems, it's fair enough IMO.